spaceship/lib/spaceship/connect_api/models/app.rb

```diff
@@ -311,6 +312,21 @@ def update_beta_group(beta_group, public_link_enabled: nil, public_link_limit: n
         Spaceship::ConnectAPI::Models.parse(resps.body)
       end
 
+      #
+      # End User License Agreements
+      #
+
+      def create_end_user_license_agreement(attributes, territory_ids: nil)
```

Comment:
Doesn't this method belong more in `spaceship/lib/spaceship/connect_api/models/end_user_license_agreement.rb`? 🤔

Reply:
Or is ti because you also need `post_end_user_license_agreement` in the Tunes section?

Reply:
Unfortunately it can't be put there, because it requires the app identifier. You can take a look at how similar classes have been implemented in fastlane before; usually, the implementation is that you'll put the `update` and `delete!` methods on the object itself (and you can do that, because you only need the object's id), and the `create` and `fetch` methods on the object's parent in the REST hierarchy (because you need the parent's id to create and retrieve it).

Reply:
Ah, okay. I was mainly looking at the app screenshot but that works with app screenshot sets. I guess that is why the create method is a class method of `AppScreenshot` in that case.